python/pylibvw.cc

```diff
@@ -857,6 +857,47 @@ uint32_t ex_get_cbandits_class(example_ptr ec, uint32_t i) { return ec->l.cb.cos
 float ex_get_cbandits_probability(example_ptr ec, uint32_t i) { return ec->l.cb.costs[i].probability; }
 float ex_get_cbandits_partial_prediction(example_ptr ec, uint32_t i) { return ec->l.cb.costs[i].partial_prediction; }
 
+const char* ex_get_ccb_type(example_ptr ec)
+{
+  switch(ec->l.conditional_contextual_bandit.type)
+  {
+    case CCB::example_type::shared:
+      return "shared";
+    case CCB::example_type::action:
+      return "action";
+    case CCB::example_type::slot:
+      return "slot";
+    default:
+      return "unset";
+  }
+}
+float ex_get_ccb_cost(example_ptr ec)
+{
+  auto* outcome_ptr = ec->l.conditional_contextual_bandit.conditional_contextual_bandit_outcome;
+  return outcome_ptr == nullptr ? 0.f : outcome_ptr->cost;
```

Comment:
If the outcome doesn't exist, Python should probably receive None instead of some default possible valid value